packages/create-project/src/helpers/install.test.ts

```diff
@@ -0,0 +1,26 @@
+import test from 'ava';
+import { replaceTemporalVersion } from './install.js';
+import { URL } from 'url';
+import { readFile, copyFile, rm } from 'fs/promises';
+import path from 'path';
+
+test('replaceTemporalVersion according to configured level', async (t) => {
+  console.log('debug windows');
+  console.log('import.meta.url', import.meta.url);
+  console.log('URL', new URL('../../test', import.meta.url));
+  const { pathname: testDir } = new URL('../../test', import.meta.url);
+  const packageJson = path.join(testDir, 'package.json');
+  console.log('packageJson', packageJson);
+  const packageJsonBackup = path.join(testDir, 'package.json.bak');
```

Comment:
I'd just create a temporary directory and not mess with the repo's structure.
We do this in other tests IIRC.